dev-tools/jenkins_setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+#!/usr/bin/env bash
+set -euox pipefail
```

Comment:
If this is going to be used a library by other scripts then I would omit this and the hashbangs.

Reply:
removed